code/datums/components/uplink.dm

```diff
@@ -147,7 +147,7 @@ GLOBAL_LIST_EMPTY(uplinks)
 							is_inaccessible = 0
 					if(is_inaccessible)
 						continue
-				if(I.restricted_species.len)
+				if(I.restricted_species != null)
```

Comment:
```suggestion
				if(length(I.restricted_species))
```

Reply:
Has the same end result in just as many characters.

Reply:
Also it should be LAZYLEN if you expect the list to ever be null.

Reply:
```suggestion
				if(I.restricted_species)
```

lol

Reply:
oh yeah that's a thing that works, thanks cobby